packages/datadog-plugin-prisma/test/index.spec.js

```diff
@@ -0,0 +1,262 @@
+'use strict'
+
+const fs = require('fs/promises')
+const path = require('path')
+const agent = require('../../dd-trace/test/plugins/agent')
+const { execSync } = require('node:child_process')
+const { expectedSchema, rawExpectedSchema } = require('./naming')
+const { ERROR_MESSAGE, ERROR_TYPE, ERROR_STACK } = require('../../dd-trace/src/constants')
+const { unlink } = require('node:fs')
+
+async function generatePrismaClient (version) {
+  const generatedClientPath = path.join(
+    __dirname,
+    `../../../versions/@prisma/client@${version}/node_modules/.prisma/client`
+  )
+  const generatedSchemaPath = path.join(__dirname, './schema.prisma')
+
+  const prismaBin = path.join(__dirname, '/../../../versions/prisma/node_modules/.bin/prisma')
+
+  const schema = `
+  generator client {
+    provider = "prisma-client-js"
+    output   = "${generatedClientPath}"
+  }
+  
+  datasource db {
+    provider = "postgresql"
+    url      = "postgres://postgres:postgres@localhost:5432/postgres"
+  }
+
+  model User {
+  id    Int     @id @default(autoincrement())
+  email String  @unique
+  name  String?
+}
+    `
+  await fs.writeFile(generatedSchemaPath, schema)
+
+  execSync(`${prismaBin} generate --schema="${generatedSchemaPath}"`, { stdio: 'inherit' })
+}
+
+function cleanUpPrismaClient () {
+  const generatedSchemaPath = path.join(__dirname, './schema.prisma')
+  unlink(generatedSchemaPath, () => { })
+}
+
+describe('Plugin', () => {
+  let prisma
+  let prismaClient
+  let tracingHelper
+
+  describe('prisma', () => {
+    withVersions('prisma', ['@prisma/client'], version => {
```

Comment:
```suggestion
    withVersions('prisma', ['@prisma/client'], version => {
      if (version === '6.1.0') return; // temp test
```

Reply:
(this won't actually get merged, just a test run)

basically run it like this in CI to test if the issue really is with 6.1.0 or if it's something to do with leaky state between test runs

Reply:
This issue has been fixed